Module/Common/Functions.XccdfXml.ps1

```diff
@@ -355,6 +362,9 @@ function Split-BenchmarkId
         'PowerPoint',
         'Word'
     )
+    $McAfeeVariations = @(
```

Comment:
This variable isn't used, would suggest removing it.  If it needs to stay, future use, use camel casing "mcAfeeVariations"